src/SprykerEco/Zed/Unzer/Business/Payment/Filter/UnzerEnabledPaymentMethodFilter.php

```diff
@@ -137,9 +223,44 @@ protected function filterEnabledPaymentMethods(
      */
     protected function getPaymentMethodKeys(PaymentMethodsTransfer $paymentMethodsTransfer): array
     {
-        return array_map(function (PaymentMethodTransfer $paymentMethodTransfer) {
-            return $paymentMethodTransfer->getPaymentMethodKey();
-        }, (array)$paymentMethodsTransfer->getMethods());
+        $paymentMethodKeys = [];
+
+        foreach ($paymentMethodsTransfer->getMethods() as $paymentMethodTransfer) {
+            $paymentMethodKeys[] = $paymentMethodTransfer->getPaymentMethodKeyOrFail();
+        }
+
+        return $paymentMethodKeys;
+    }
+
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     *
+     * @return \Generated\Shared\Transfer\UnzerKeypairTransfer
+     */
+    protected function getStandardUnzerKeypair(QuoteTransfer $quoteTransfer): UnzerKeypairTransfer
+    {
+        return $this->getUnzerKeypair(
+            $quoteTransfer->getStoreOrFail(),
```

Comment:
please list the required transfer properties at the affected API items

Reply:
Is already part of specification.